src/dmd/dmodule.d

```diff
@@ -400,7 +400,7 @@ extern (C++) final class Module : Package
     Identifiers* versionidsNot; // forward referenced version identifiers
 
     Macro* macrotable;          // document comment macros
-    Escape escapetable;        // document comment escapes
+    Escape* escapetable;        // document comment escapes
```

Comment:
IIRC, the C++ version used `Escape **`, can you update it to `Escape*` as well?

Reply:
No, it was already good, see https://github.com/dlang/dmd/pull/9418/files/50d758aa1ce8f7f5d073a357da5acbcc88171b40#diff-5c1e8d0f4cb0f0151c3b5d3dc6dce89f